simulator/mips/_instr.h

```diff
@@ -0,0 +1,85 @@
+enum instr_size_and_shifts {
```

Comment:
Please add header comment with copyright info – see other files as examples.
Also, could you please rename file to something more self-explanationary, like "mips_instr_decode.h"?